### PR TITLE
issue 668 navigation to "help" may need adjustment

### PR DESCRIPTION
**Pull request by zawmoelwin**
Take option 2 as solution. Rename the "Contributors" page to "Contribute"
change route contributors to contribute but keep the contributors_path, change the locale en "navigaiton.contributors"  to "Contribute" in footer section

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (julianguyen) [inline .env]: Please remove this file from the PR :)

File: config/locales/en.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@ en:
  navigation:
    about: 'About'
    blog: 'Blog'
-   contributors: 'Contributors'
+   contributors: 'Contribute'
```
Comment (julianguyen) [inline config/locales/en.yml]: If we're going to take this "Contribute" route, let's also rename the key to "contribute"

File: config/routes.rb [hunk 3]
```diff
@@ -81,8 +81,8 @@
 
   get 'pages/home'
   match 'about', to: 'pages#about', via: :get
-  match 'contributors', to: 'pages#contributors', via: :get
-  match 'partners', to: 'pages#partners', via: :get
+  match 'contribute', to: 'pages#contributors', via: :get , as: 'contributors'
```
Comment (julianguyen) [inline config/routes.rb]: We might as well rename the page#contributors to Contribute

File: db/schema.rb [hunk 4]
```diff
@@ -234,8 +234,8 @@
     t.datetime "invitation_sent_at"
     t.datetime "invitation_accepted_at"
     t.integer  "invitation_limit"
-    t.integer  "invited_by_id"
     t.string   "invited_by_type"
+    t.integer  "invited_by_id"
```
Comment (julianguyen) [inline db/schema.rb]: Please remove this file from the PR.

Comment (julianguyen) [reply db/schema.rb]: I can't comment on the package-lock.json file for some reason, but please also remove that file too.

File: config/locales/en.yml (hunk 2 above)
Comment (julianguyen) [reply config/locales/en.yml]: We'll also need to translate this in all of our locales as well.

File: db/schema.rb (hunk 4 above)
Comment (zawmoelwin) [reply db/schema.rb]: noted.

File: config/locales/en.yml (hunk 2 above)
Comment (zawmoelwin) [reply config/locales/en.yml]: I will edit the key in all locales but we need to open another issue for translation.

**Comment by zawmoelwin** [comment]
@julianguyen 
I can't remove the package.json as it is the config file for yarn install. Thank you for the warm welcome. In this update,  
-Remove .env
-Remove schema.rb
-Change the key navigation:contributors to navigation:contribute
-Change the page.contributors key to page.contribute
-Change the routes from contributors to contribute.
-Change rspec  test files from get: contributors to get: contribute.

**Comment by camillevilla** [comment]
@julianguyen: just a heads up, @zawmoelwin and I are going to do some pair programming tomorrow (10/7) to rebase and clean up commit messages

**Pull request closed without merging**